app-containers/podman/podman-4.6.2.ebuild

```
@@ -0,0 +1,135 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit go-module tmpfiles linux-info
+# inherit flag-o-matic
+
+DESCRIPTION="A tool for managing OCI containers and pods with Docker-compatible CLI"
+HOMEPAGE="https://github.com/containers/podman/ https://podman.io/"
+if [[ ${PV} == *9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/containers/podman.git"
+else
+	SRC_URI="https://github.com/containers/podman/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm64"
+fi
+LICENSE="Apache-2.0 BSD BSD-2 CC-BY-SA-4.0 ISC MIT MPL-2.0"
+SLOT="0"
+IUSE="apparmor btrfs cgroup-hybrid docker +fuse +init +rootless +seccomp selinux systemd"
+RESTRICT="test"
+
+RDEPEND="
+	app-crypt/gpgme:=
+	>=app-containers/conmon-2.0.0
+	>=app-containers/containers-common-0.56.0
+	dev-libs/libassuan:=
+	dev-libs/libgpg-error:=
+	sys-apps/shadow:=
+
+	apparmor? ( sys-libs/libapparmor )
+	btrfs? ( sys-fs/btrfs-progs )
+	cgroup-hybrid? ( >=app-containers/runc-1.0.0_rc6  )
+	!cgroup-hybrid? ( app-containers/crun )
+	docker? ( !app-containers/docker-cli )
+	fuse? ( sys-fs/fuse-overlayfs )
+	init? ( app-containers/catatonit )
+	rootless? ( app-containers/slirp4netns )
+	seccomp? ( sys-libs/libseccomp:= )
+	selinux? ( sec-policy/selinux-podman sys-libs/libselinux:= )
+	systemd? ( sys-apps/systemd:= )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	dev-go/go-md2man
+"
+
+CONFIG_CHECK="
+	~USER_NS
+"
+
+pkg_setup() {
+	use btrfs && CONFIG_CHECK+=" ~BTRFS_FS"
+	linux-info_pkg_setup
+}
+
+src_prepare() {
+	default
+	for file in apparmor_tag btrfs_installed_tag btrfs_tag selinux_tag systemd_tag; do
+		[[ -f hack/"${file}".sh ]] || die
+	done
+
+	for feature in apparmor selinux systemd; do
+		if use "${feature}"; then
+			echo -e "#!/usr/bin/env bash\n echo ${feature}" > hack/"${feature}"_tag.sh || die
+		else
+			echo -e "#!/usr/bin/env bash\n true" > hack/"${feature}"_tag.sh || die
+		fi
+	done
+
+	if use btrfs; then
+		echo -e "#!/usr/bin/env bash\n true" | tee hack/btrfs_installed_tag.sh hack/btrfs_tag.sh || die
+	else
+		echo -e "#!/usr/bin/env bash\n echo exclude_graphdriver_btrfs" > hack/btrfs_installed_tag.sh || die
+		echo -e "#!/usr/bin/env bash\n echo btrfs_noversion" > hack/btrfs_tag.sh || die
+	fi
+
+	use seccomp || eapply "${FILESDIR}"/disable-seccomp-4.6.2.patch
+}
+
+src_compile() {
+	# Filter unsupported linker flags
+	# TODO: Is this really needed? If yes then please uncomment.
+	# filter-flags '-Wl,*'
+
+	# TODO I dont understand why this is needed. Uncomment if something goes wrong
+	# export -n GOCACHE GOPATH XDG_CACHE_HOME
+
+	export PREFIX="${EPREFIX}/usr"
+	emake BUILDFLAGS="-v -work -x" GOMD2MAN="go-md2man"
+	use docker && emake BUILDFLAGS="-v -work -x" GOMD2MAN="go-md2man" docker-docs
+
+}
+
+src_install() {
+	default
```

What if we delete these 3 lines and replace with:
```suggestion
	emake DESTDIR="${D}" install install.completions $(usev docker install.docker-full)
```